tools/gen_addon_readme.py

```diff
@@ -170,8 +170,8 @@ def generate_fragment(org_name, repo_name, branch, addon_name, file):
     return fragment
 
 
-def gen_one_addon_readme(
-        org_name, repo_name, branch, addon_name, addon_dir, manifest):
+def gen_one_addon_readme(org_name, repo_name, branch, addon_name, addon_dir,
+                         manifest, template_filename):
```

Comment:
You can just use a default argument here:

```suggestion
def gen_one_addon_readme(org_name, repo_name, branch, addon_name, addon_dir,
                         manifest, template_filename=os.path.join(os.path.dirname(__file__), 'gen_addon_readme.template')):
```

Reply:
Changed default value on  @click option

Reply:
I'm pretty sure the hanging indentation is not pylint friendly.
Is this code covered by the same standards as Odoo module repos.

Reply:
No, it has its own standards AFAICS: https://github.com/OCA/maintainer-tools/blob/e3b46970af62cbbf99f0b45a6675eb4cae744d6d/.pre-commit-config.yaml#L4-L9

When somebody adds black there, then all will be consistent. But that seems a separate issue, as the rest of the repo is already not black-friendly.